MedvedevKS/lab_2/lab_2.cpp

```diff
@@ -0,0 +1,60 @@
+#include "stdafx.h"
+#include "iostream"
+#include "conio.h"
+using namespace std;
+
+
+int main()
+{
+	const int size(11);
+	const int size1 = size / 2 + size % 2;
+	int random_arr[size], odd_arr[size1];
+	cout << "Generated 1st array: \t\t";
+	int i(0);
+	for (i; i < size; i++)
```

Comment:
```suggestion
	for (int i(0); i < size; i++)
```
Do not decalre iterative vars outside of loop